src/SFA.DAS.LoginService.Application/ChangeEmail/StartChangeEmail/StartChangeEmailRequest.cs

```diff
@@ -1,11 +1,13 @@
 using MediatR;
 using System;
+using System.Security.Claims;
 
 namespace SFA.DAS.LoginService.Application.ChangeEmail.StartChangeEmail
 {
     public class StartChangeEmailRequest : IRequest<StartChangeEmailResponse>
     {
         public Guid ClientId { get; set; }
+        public string SubjectId { get; set; }
```

Comment:
I can see the claim is called Sub and this is where we store the Email address, but do we need to call it SubjectId? And do we need the property

public string CurrentEmailAddress { get; set; }

Are they not the same thing?

Reply:
No, it's really `UserId`.  I've changed it 👍 

You're right about `CurrentEmailAddress` as well, it shouldn't be in the request.